content/documentation/rpc/wallet/multisig/create_signature_share.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: wallet/multisig/createSignatureShare
+description: RPC Multisig | Iron Fish Documentation
+---
+
+Creates a signature share for a participant for a given signing package. After this step, the signatures are collected and used by the [wallet/multisig/aggregateSignatureShares](/developers/documentation/rpc/wallet/multisig/aggregate_signature_shares) to create the final signed transaction.
```

Comment:
nit - seems like this is missing a word: `the ... endpoint to create...` or `the ... RPC to create...`